Utils/Utils/String/StringBuilder.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  StringBuilder.swift
+//  Utils
+//
+//  Created by Alexander Filimonov on 28/06/2019.
+//  Copyright © 2019 Surf. All rights reserved.
+//
+
+import Foundation
+
+public class StringBuilder {
+
+    // MARK: - Nested types
+
+    class StringPart {
+        let string: String
+        let attributes: [StringAttribute]
+        var range: NSRange?
+
+        init(string: String, attributes: [StringAttribute] = []) {
+            self.string = string
+            self.attributes = attributes
+        }
+    }
+
+    // MARK: - Readonly properties
+
+    public var value: NSMutableAttributedString {
+        return renderAttributedString()
+    }
+
+    // MARK: - Private properties
+
+    private var parts: [StringPart] = []
+    private var globalAttributes: [StringAttribute] = []
+
+    // MARK: - Initialization
+
+    public init(attributes: [StringAttribute] = []) {
```

Comment:
Maybe "globalAttributes" will be more clear?

Reply:
Yes, made it